website/docs/r/cloud_aws_integrations.html.markdown

```diff
@@ -307,6 +309,67 @@ resource "newrelic_cloud_aws_integrations" "bar" {
+* `elasticache` - (Optional) AWS ElastiCache. See [Integration blocks](#integration-blocks) below for details.
+* `elasticbeanstalk` - (Optional) AWS Elastic Beanstalk. See [Integration blocks](#integration-blocks) below for details.
+* `elasticsearch` - (Optional) AWS ElasticSearch. See [Integration blocks](#integration-blocks) below for details.
+* `elb` - (Optional) AWS ELB (Classic). See [Integration blocks](#integration-blocks) below for details.
+* `emr` - (Optional) AWS EMR. See [Integration blocks](#integration-blocks) below for details.
+* `health` - (Optional) AWS Health. See [Integration blocks](#integration-blocks) below for details.
+* `iam` - (Optional) AWS IAM. See [Integration blocks](#integration-blocks) below for details.
+* `iot` - (Optional) AWS IoT. See [Integration blocks](#integration-blocks) below for details.
+* `kinesis_firehose` - (Optional) Amazon Kinesis Data Firehose. See [Integration blocks](#integration-blocks) below for details.
+* `lambda` - (Optional) AWS Lambda. See [Integration blocks](#integration-blocks) below for details.
+* `rds` - (Optional) Amazon RDS. See [Integration blocks](#integration-blocks) below for details.
+* `redshift` - (Optional) Amazon Redshift. See [Integration blocks](#integration-blocks) below for details.
+* `route53` - (Optional) Amazon Route 53. See [Integration blocks](#integration-blocks) below for details.
+* `s3` - (Optional) Amazon S3. See [Integration blocks](#integration-blocks) below for details.
+* `ses` - (Optional) Amazon SES. See [Integration blocks](#integration-blocks) below for details.
+* `sns` - (Optional) AWS SNS. See [Integration blocks](#integration-blocks) below for details.
+* `sqs` - (Optional) AWS SQS. See [Integration blocks](#integration-blocks) below for details.
+* `x_ray` - (Optional) AWS X-Ray. See [Integration blocks](#integration-blocks) below for details.
+x
+
+The following arguments are supported with minimum metrics polling interval of 900 seconds
+
+* `ebs` - (Optional) Amazon EBS. See [Integration blocks](#integration-blocks) below for details.
+* `kinesis` - (Optional) AWS Kinesis. See [Integration blocks](#integration-blocks) below for details.
+
+The following arguments are supported with minimum metrics polling interval of 3600 seconds
+
+* `billing` - (Optional) AWS Billing. See [Integration blocks](#integration-blocks) below for details.
+* `trusted_advisor` - (Optional) AWS Trusted Advisor. See [Integration blocks](#integration-blocks) below for details.
+
 All other arguments are dependent on the services to be integrated, which have been listed in the collapsible section below. All of these are **optional** blocks that can be added in any required combination. **For details on arguments that can be used with each service, check out the [`Integration` blocks](#integration-blocks) section below.**
 <details>
   <summary>Expand this section to view all supported AWS services supported, that may be integrated via this resource.</summary>
```

Comment:
@abasha1234 a tiny request - can you please move the contents of this section (the table comprising a list of all supported integrations, in this summary section) to a different heading before "Argument Reference"? 

i.e. delete this <details> and <summary> block and above the "Argument Reference" section, let's add something like

```
## Supported AWS Integrations
<details>
<summary>Expand this list to view AWS integrations/services which can be configured with New Relic via this resource.</summary>

| Block                   | Description                   |
|-------------------------|-------------------------------|
| `alb`                   | ALB Integration               |
| `api_gateway`           | API Gateway Integration       |
| `auto_scaling`          | Auto Scaling Integration      |
| `aws_app_sync`          | AppSync Integration           |
| `aws_athena`            | Athena Integration            |
| `aws_cognito`           | Cognito Integration           |
| `aws_connect`           | Connect Integration           |
| `aws_direct_connect`    | Direct Connect Integration    |
| `aws_fsx`               | FSx Integration               |
| `aws_glue`              | Glue Integration              |
| `aws_kinesis_analytics` | Kinesis Analytics Integration |
| `aws_media_convert`     | MediaConvert Integration      |
| `aws_media_package_vod` | Media Package VOD Integration |
| `aws_mq`                | MQ Integration                |
| `aws_msk`               | MSK Integration               |
| `aws_neptune`           | Neptune Integration           |
| `aws_qldb`              | QLDB Integration              |
| `aws_route53resolver`   | Route53 Resolver Integration  |
| `aws_states`            | States Integration            |
| `aws_transit_gateway`   | Transit Gateway Integration   |
| `aws_waf`               | WAF Integration               |
| `aws_wafv2`             | WAFv2 Integration             |
| `billing`               | Billing Integration           |
| `cloudfront`            | CloudFront Integration        |
| `cloudtrail`            | CloudTrail Integration        |
| `doc_db`                | DocumentDB Integration        |
| `dynamodb`              | DynamoDB Integration          |
| `ebs`                   | EBS Integration               |
| `ec2`                   | EC2 Integration               |
| `ecs`                   | ECS Integration               |
| `efs`                   | EFS Integration               |
| `elasticache`           | ElastiCache Integration       |
| `elasticbeanstalk`      | Elastic Beanstalk Integration |
| `elasticsearch`         | Elasticsearch Integration     |
| `elb`                   | ELB Integration               |
| `emr`                   | EMR Integration               |
| `health`                | Health Integration            |
| `iam`                   | IAM Integration               |
| `iot`                   | IoT Integration               |
| `kinesis`               | Kinesis Integration           |
| `kinesis_firehose`      | Kinesis Firehose Integration  |
| `lambda`                | Lambda Integration            |
| `rds`                   | RDS Integration               |
| `redshift`              | Redshift Integration          |
| `route53`               | Route53 Integration           |
| `s3`                    | S3 Integration                |
| `ses`                   | SES Integration               |
| `sns`                   | SNS Integration               |
| `sqs`                   | SQS Integration               |
| `trusted_advisor`       | Trusted Advisor Integration   |
| `vpc`                   | VPC Integration               |
| `x_ray`                 | X-Ray Integration             |
</details>
```